packages/atlas/src/components/_comments/Comment/Comment.tsx

```diff
@@ -1,336 +1,305 @@
-          >
-            {loading ? (
-              <div>
-                <SkeletonLoader width={128} height={20} bottomSpace={8} />
-                <SkeletonLoader width="100%" height={16} bottomSpace={8} />
-                <SkeletonLoader width="70%" height={16} />
-              </div>
-            ) : (
-              <div>
-                <CommentHeader isDeleted={isDeleted}>
-                  <StyledLink to={memberUrl || ''}>
-                    <Text variant="h200" margin={{ right: 2 }}>
-                      {memberHandle}
-                    </Text>
-                  </StyledLink>
-                  <CommentHeaderDot />
-                  <Tooltip text={tooltipDate} placement="top" offsetY={4} delay={[1000, null]}>
-                    <StyledLink to={absoluteRoutes.viewer.video(videoId, { commentId: id })}>
-                      <HighlightableText variant="t200" secondary margin={{ left: 2, right: 2 }}>
-                        {formatDateAgo(createdAt || new Date())}
-                      </HighlightableText>
-                    </StyledLink>
-                  </Tooltip>
-                  {isEdited && !isDeleted && (
-                    <>
-                      <CommentHeaderDot />
-                      <HighlightableText
-                        variant="t200"
-                        secondary
-                        margin={{ left: 2 }}
-                        onClick={() => onEditLabelClick?.()}
-                      >
-                        edited
-                      </HighlightableText>
-                    </>
-                  )}
-                </CommentHeader>
-                {isDeleted ? (
-                  <DeletedComment variant="t200" color={cVar('colorTextMuted')}>
-                    <StyledSvgActionTrash /> Comment deleted by the {isModerated ? 'channel owner' : 'author'}
-                  </DeletedComment>
-                ) : (
-                  <CommentBody>{text}</CommentBody>
-                )}
-              </div>
-            )}
-          </CSSTransition>
-        </SwitchTransition>
-        <ContextMenu
-          placement="bottom-end"
-          disabled={loading || !shouldShowKebabButton}
-          items={contexMenuItems}
-          trigger={
-            <KebabMenuIconButton
-              icon={<SvgActionMore />}
-              variant="tertiary"
-              size="small"
-              isActive={shouldShowKebabButton}
-            />
+    const handleOpenSignInDialog = () => !activeMemberId && openSignInDialog({ onConfirm: signIn })
+
+    const replyAvatars = comment?.replies?.map((comment) => ({
+      url:
+        comment?.author.metadata.avatar?.__typename === 'AvatarUri' ? comment?.author.metadata.avatar?.avatarUri : '',
+      handle: comment.author.handle,
+    }))
+
+    const loading = loadingQuery || !commentId
+
+    const userReactions = comment?.reactions
+      .filter((reaction) => reaction.memberId === activeMemberId)
```

Comment:
The query should apply this filter already, do we need to `.filter` here as well?

Reply:
yep